### DISCUSSION TITLE
Issues with Github desktop

### DISCUSSION THREAD

**Discussion by AdnanRaashid** [community/community#52375] [New to GitHub] [answered]
Hello,
I am currently having issues authorizing my Github desktop, it comes to the point I have to click the green 'Authorize desktop' button after going through a browser, but that button remains inactive and not responding to clicking.
someone kindly assist me please. 

![IMG_20230409_162056_062](https://user-images.githubusercontent.com/120261421/230781423-ad3572f7-3b1a-424c-ab42-59c475a4207c.jpg)

**Comment by neutrinomuon** [comment, marked as answer]
Hi,

I'm sorry you're having trouble authorizing your GitHub Desktop. As far as I know, there could be several reasons why the "Authorize desktop" button is not working when you click on it. From your image, are you using Internet Explorer? This could be one of the reasons. From GitHub webpages, I see it recommends using a modern web browser, such as Chrome, Firefox, or Edge, for the best experience.

I hope this helps. Please let me know more details and whether it works.

**Reply by TheRagingCasuals** [reply]
Thank you very much. ****

**Comment by AdnanRaashid** [comment]
Yes, thanks.
It did work on Edge.

**Reply by neutrinomuon** [reply]
Great! Can you please mark the question as answered? It would be really helpful for others in the community. Thank you so much

**Comment by AdnanRaashid** [comment]
I'm finding it difficult to get back to that page, could you send a direct
link please?

**Reply by neutrinomuon** [reply]
Do you mean the link to our discussion? https://github.com/orgs/community/discussions/52375

**Comment by hrada777** [comment]
> Hello,
> I am currently having issues authorizing my Github desktop, it comes to the point I have to click the green 'Authorize desktop' button after going through a browser, but that button remains inactive and not responding to clicking.
> someone kindly assist me please. 
> 
> 
> ![IMG_20230409_162056_062](https://user-images.githubusercontent.com/120261421/230781423-ad3572f7-3b1a-424c-ab42-59c475a4207c.jpg)

Ya
